src/aspire/commands/preprocess.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+import click
+
+from aspire.estimation.noise import WhiteNoiseEstimator
+from aspire.source.relion import RelionSource
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.option('--data_folder', default=None,
+              help='Path to data folder')
+@click.option('--starfile_in', required=True,
+              help='Path to input starfile relative to data folder')
+@click.option('--starfile_out', required=True,
+              help='Path to output starfile relative to data folder')
+@click.option('--pixel_size', default=1, type=float,
+              help='Pixel size of images in starfile')
+@click.option('--max_rows', default=None, type=int,
+              help='Max. no. of image rows to read from starfile')
+@click.option('--flip_phase', default=True,
+              help='perform phase flip or not')
+@click.option('--max_resolution', default=16, type=int,
+              help='Resolution of downsampled images read from starfile')
+@click.option('--normalize_background', default=True,
```

Comment:
Whether to normalize images to background noise

Reply:
Done in commit 0b58b0d.